src/utils/__tests__/lesson-metadata.test.ts

```diff
@@ -1,135 +1,167 @@
+    const graphqlMetadata = {
+      collection: {title: 'Xtreme React Course', lessons: [4, 5, 6]},
+    } as LessonResource
 
-test('graphql collection is used if Sanity collection is not present', () => {
-  const graphqlMetadata = {
-    collection: {title: 'Xtreme React Course', lessons: [4, 5, 6]},
-  } as LessonResource
+    // a collection needs to include some course metadata and a non-empty list of
+    // lessons to be considered present.
+    const sanityMetadata = {
+      collection: {title: 'Ultimate React Course', lessons: [1, 2, 3]},
+    } as LessonResource
 
-  // a collection needs to include some course metadata and a non-empty list of
-  // lessons to be considered present.
-  const sanityMetadata = {
-    collection: {title: 'Ultimate React Course', lessons: [] as Array<{}>},
-  } as LessonResource
+    const expectedResult = {
+      collection: {title: 'Ultimate React Course', lessons: [1, 2, 3]},
+    }
 
-  const expectedResult = {
-    collection: {title: 'Xtreme React Course', lessons: [4, 5, 6]},
-  }
+    const result = mergeLessonMetadata(graphqlMetadata, sanityMetadata)
 
-  const result = mergeLessonMetadata(graphqlMetadata, sanityMetadata)
+    expect(result).toEqual(expectedResult)
+  })
+
+  test('graphql collection is used if Sanity collection is not present', () => {
+    const graphqlMetadata = {
+      collection: {title: 'Xtreme React Course', lessons: [4, 5, 6]},
+    } as LessonResource
+
+    // a collection needs to include some course metadata and a non-empty list of
+    // lessons to be considered present.
+    const sanityMetadata = {
+      collection: {title: 'Ultimate React Course', lessons: [] as Array<{}>},
+    } as LessonResource
+
+    const expectedResult = {
+      collection: {title: 'Xtreme React Course', lessons: [4, 5, 6]},
+    }
+
+    const result = mergeLessonMetadata(graphqlMetadata, sanityMetadata)
+
+    expect(result).toEqual(expectedResult)
+  })
+})
 
-  expect(result).toEqual(expectedResult)
+describe('deriveDataFromBaseValues()', () => {
+  test('it returns an empty object if path is blank', () => {
+    expect(deriveDataFromBaseValues({path: undefined})).toEqual({})
+  })
+
+  test('it returns URLs when path is set', () => {
+    const expectedResult = {
+      http_url: expect.stringContaining('/lessons/some-slug'),
+      lesson_view_url: expect.stringContaining(
+        '/api/v1/lessons/some-slug/views',
+      ),
+      download_url: expect.stringContaining(
+        '/api/v1/lessons/some-slug/signed_download',
+      ),
+    }
+
+    const result = deriveDataFromBaseValues({path: '/lessons/some-slug'})
+
+    expect(result).toMatchObject(expectedResult)
+  })
+
+  test('it throws an error when there is no leading slash', () => {
+    expect(() => {
+      deriveDataFromBaseValues({path: 'lessons/some-slug'})
+    }).toThrow(
+      'Invariant failed: Path value must begin with a forward slash (`/`).',
+    )
+  })
```

Comment:
Looks like your build is failing due to this test. It’s expecting a different message than its receiving. 

This occurs since invariant minifies its messages in production. See the note here https://www.npmjs.com/package/invariant

Reply:
good catch! Fixed it by loosen the expectation a little

![CleanShot 2022-08-16 at 17 02 18@2x](https://user-images.githubusercontent.com/694063/184993111-d2933fd7-1d59-499c-b0ab-8ac8a0aeeeda.png)